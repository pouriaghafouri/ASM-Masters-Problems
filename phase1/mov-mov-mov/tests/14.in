-6048718003998904249 1404762273848593534
