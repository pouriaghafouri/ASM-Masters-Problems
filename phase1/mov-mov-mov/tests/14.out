-7453480277847497783
